sdk/keyvault/azure-security-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/implementation/signature/KeyVaultKeyLessRsaSignatureTest.java

```diff
@@ -102,4 +102,9 @@ public void setDigestNameAndEngineSignTest() throws InvalidAlgorithmParameterExc
         assertArrayEquals("fakeValue".getBytes(), keyVaultKeyLessRsaSignature.engineSign());
     }
 
+    @Test
+    public void engineSetParameterWithNullParameterTest() {
```

Comment:
Do we need all these 3 test cases:
1. algorithmParameterSpec= null;
2. algorithmParameterSpec instanceof PSSParameterSpec == false
3. ((PSSParameterSpec) params).getDigestAlgorithm() == null

Reply:
`((PSSParameterSpec) params).getDigestAlgorithm()` cannot be empty because there is a [check](https://github.com/openjdk/jdk11u-dev/blob/90d529355b0fa36ca3378efae8dc6ba4b5203869/src/java.base/share/classes/java/security/spec/PSSParameterSpec.java#L139), but I think we should consider the case where `signatureParameters.getDigestAlgorithm();` is incorrect.